TestTicTacToe/InputValidatorTest.cs

```diff
@@ -0,0 +1,37 @@
+using TicTacToe;
+using Xunit;
+
+namespace TestTicTacToe
+{
+
+    public class InputValidatorTest
+    {
+        [Theory]
+        [InlineData("3,3")]
+        [InlineData("1,1")]
+        [InlineData("2,2")]
+        [InlineData("1,3")]
+        public void Should_Equal_True_If_Input_Is_Valid(string input)
```

Comment:
I like that you have used a Theory here, however it is quite difficult to see what a valid input is. It might be better to do all the valid inputs eg 1,1 1,2 1,3 2,1, 2,2 2,3......